src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/TaxInformationList.java

```diff
@@ -12,15 +12,15 @@
 import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
 
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(name = "", propOrder = { "taxInformation" })
+@XmlType(name = "", propOrder = { "taxInformations" })
 @XmlRootElement(name = "TaxInformationList")
 public class TaxInformationList {
 
     @XmlAttribute(name = "isChanged")
     @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
     private String isChanged;
     @XmlElement(name = "TaxInformation")
-    private List<TaxInformation> taxInformation;
+    private List<TaxInformation> taxInformations;
```

Comment:
Another list property that might work out better with a name suffix of "Items" or "Entries" or "Details".

Reply:
switched to taxInformationDetails